internal/login/login_request.go

```diff
@@ -27,6 +27,10 @@ func loginRequest(method string, url string, payload io.Reader) (loginRequestRes
 func loginRequestWithHeaders(method string, url string, payload io.Reader, headers []loginHeader) (loginRequestResponse, error) {
 	req, err := request.NewRequest(method, url, payload)
 
+	if err != nil {
+		return loginRequestResponse{}, nil
```

Comment:
This should be `return loginRequestResponse{}, err`, to surface the error instead of nil